assets/js/controllers/send.controller.js

```diff
@@ -179,6 +186,11 @@ function SendCtrl ($scope, $log, Wallet, Alerts, currency, $uibModal, $uibModalI
       .then(transactionSucceeded).catch(transactionFailed);
   };
 
+  $scope.sendInputMetrics = (metric) => {
+    let root = $rootScope.rootURL ? $rootScope.rootURL : '/';
+    $http.get(`${root}event?name=wallet_web_tx_from_${metric}`);
```

Comment:
It is the same endpoint. I don't know if I would consider this the responsibility of My-Wallet-V3 though, since the events are specific to the web wallet.